sklearn/datasets/tests/test_svmlight_format.py

```diff
@@ -553,3 +553,22 @@ def test_load_offset_exhaustive_splits():
 def test_load_with_offsets_error():
     with pytest.raises(ValueError, match="n_features is required"):
         _load_svmlight_local_test_file(datafile, offset=3, length=3)
+
+
+def test_multilabel_y_explicit_zeros(tmp_path):
+    save_path = str(tmp_path / "svm_explicit_zero")
+    rng = np.random.RandomState(42)
+    X = rng.randn(3, 5).astype(np.float64)
+    indptr = np.array([0, 2, 3, 6])
+    indices = np.array([0, 2, 2, 0, 1, 2])
+    data = np.array([0, 2, 3, 4, 5, 6])
+    y = sp.csr_matrix((data, indices, indptr), shape=(3, 3))
+
+    assert_array_equal(y.todense(), np.array([[0, 0, 2], [0, 0, 3], [4, 5, 6]]))
```

Comment:
I think this makes sense to assert this for `y_load` and not `y`. Currently the test is to make sure that `y.todense` works. Checking for `y_load` makes sure that `dump + load` returns the same data.

Reply:
This was suggested by @jjerphan for clarity, not necessarily to test `y.todense`. I've replaced this w/ comments for more clarity and focus.